HuntHelper/Managers/IpcSystem.cs

```diff
@@ -50,10 +60,29 @@ public void Dispose()
         _framework.RunOnFrameworkThread(() =>
             _trainManager.HuntTrain.Select(AsMobRecord).ToList()
         ).Result;
+    
+    private void ImportTrainList(List<MobRecord> trainList)
+    {
+        _trainManager.ImportFromIPC = true;
+        /* if we rework this and use a bool instead
+         * since train manager is used multiple times in this class, but hunt train ui is only used for this
+         * we can rework it so hunt train ui is not needed
+         * 
+         * we can then check the bool in the UI
+         * 
+         * separating things
+        */
```

Comment:
oh i just realized one thing missing from this. i ran into it in my testing, which is that if the train window isn't open then the import popup won't show up when the ipc function is called. that can be confusing. so that's why `_huntTrainUi.OpenImportPopup()` was setting
```csharp
HuntTrainWindowVisible = true;
```

Reply:
I think operationally thats fine? as the importFromIpc bool has been set to true, so as soon as the user opens the ui the import modal will show.

Reply:
If anything, I'd personally probably find it a bit annoying if one plugin forced open another's UI. But idk, that's just me.